roles/splunk_common/tasks/scloud/construct_stream.yml

```diff
@@ -0,0 +1,74 @@
+---
+- name: Define spec file
+  set_fact:
+    stream_pipeline_spec_file: "{{ splunk.home }}/etc/{{ splunk.dsp.pipeline_name }}-spec.json"
+
+- name: Generate data stream pipeline specification
+  become: yes
+  become_user: "{{ splunk.user }}"
+  block:
+  - name: Find existing spec
+    stat:
+      path: "{{ stream_pipeline_spec_file }}"
+    register: pipe_spec
+
+  - name: Read existing spec
```

Comment:
Chalking this up to limited-time-constraints-due-to-hackweek, but we should probably gather the existing spec by querying DSP instead of inspecting the file created.